python/GangaLHCb/Lib/RTHandlers/RTHUtils.py

```diff
@@ -57,12 +57,6 @@ def lhcbdiracAPI_script_template():
 def is_gaudi_child(app):
     if isType(app, Gaudi):
         return True
-
-    if isType(app, TaskApplication):
-        from GangaLHCb.Lib.Applications import GaudiPythonTask, BenderTask
-        if not isType(app, GaudiPythonTask) and not isType(app, BenderTask):
```

Comment:
Apologies I should get this but aren't these classes derived from the ```GangaGaudi``` codebase? Why is the check being removed?